composer.json

```
@@ -26,12 +26,12 @@
         "doctrine/doctrine-bundle": "^1.12.3 || ^2.0",
         "doctrine/persistence": "^1.3.4 || ^2.0",
         "sonata-project/admin-bundle": "^3.99",
-        "sonata-project/block-bundle": "^3.20",
+        "sonata-project/block-bundle": "^3.20 || ^4.0",
```

```suggestion
        "sonata-project/block-bundle": "^4.0",
```
v3 is not installed anymore